### PR TITLE
PIV-1237 - property additions for /revoke and /manage

### PR DESCRIPTION
**Pull request by crolarlibertyva**
Requires https://github.com/department-of-veterans-affairs/health-apis-informational-endpoints/pull/12

### CODE REVIEW

**Comment by beaugrantham-va** [comment]
The artifacts being produced do not include the two new endpoints. This is likely a problem with `health-apis-informational-endpoints`. I don't want to merge this until we see proper artifacts.

File: conformance-unifier/src/main/resources/application-qa.properties [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
 stu3.capability.security.token-endpoint=https://dev-api.va.gov/oauth2/token
 stu3.capability.security.authorize-endpoint=https://dev-api.va.gov/oauth2/authorization
+stu3.capability.security.management-endpoint=https://dev-api.va.gov/oauth2/manage
+stu3.capability.security.revoke-endpoint=https://dev-api.va.gov/oauth2/revoke
 
 # R4 Unified Metadata
 r4.capability.security.token-endpoint=https://dev-api.va.gov/oauth2/token
 r4.capability.security.authorize-endpoint=https://dev-api.va.gov/oauth2/authorization
+r4.capability.security.management-endpoint=https://dev-api.va.gov/oauth2/manage
+r4.capability.security.revoke-endpoint=https://dev-api.va.gov/oauth2/revoke
```
Comment (beaugrantham-va) [inline conformance-unifier/src/main/resources/application-qa.properties:17]: Some of these are `r4.capability.security.revoke-endpoint` and some are `r4.capability.security.revocation-endpoint`.

Can you take a pass through all these new properties and just double check they are right?

File: conformance-unifier/src/main/resources/application-qa.properties [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
 stu3.capability.security.token-endpoint=https://dev-api.va.gov/oauth2/token
 stu3.capability.security.authorize-endpoint=https://dev-api.va.gov/oauth2/authorization
+stu3.capability.security.management-endpoint=https://dev-api.va.gov/oauth2/manage
+stu3.capability.security.revocation=https://dev-api.va.gov/oauth2/revoke
 
 # R4 Unified Metadata
 r4.capability.security.token-endpoint=https://dev-api.va.gov/oauth2/token
 r4.capability.security.authorize-endpoint=https://dev-api.va.gov/oauth2/authorization
+r4.capability.security.management-endpoint=https://dev-api.va.gov/oauth2/manage
+r4.capability.security.revocation=https://dev-api.va.gov/oauth2/revoke
```
Comment (beaugrantham-va) [inline conformance-unifier/src/main/resources/application-qa.properties:17]: These are incorrect in a different way now.

Update and run the unifier locally testing _all environments_ (`--spring.profiles.active=xxx`) to make sure everything works to save some time and trouble.

**Comment by dbrown-va** [comment]
Changes look good now, holding approval until the `<informational.version>` can be modified to the non-SNAPSHOT version.

**Comment by beaugrantham-va** [comment]
Looks good now. Update the SNAPSHOT dependency and we'll get this merged.

**Pull request merged**